app/jobs/attached_report_job.rb

```diff
@@ -21,7 +22,7 @@ def perform args
 
     condition = conditions.map { |c| "(#{c})" }.join ' OR '
 
-    report = model.where(condition, parameters).send method_name, options
+    report = model.where(condition, parameters).reorder(order).send method_name, options
```

Comment:
¿Qué pasa cuando el orden viene para las tablas que se incluyen? Acordate que hay varios de esos en las opciones del listado, si las probaste sale el merge... sino... servite(TM).

Reply:
este order ya viene sql hermoso  `"findings"."priority" DESC, "findings"."risk" DESC, "findings"."state" ASC` con las tablas incluidas así que no debería haber problema, igual ahora pruebo alguna extra.

Reply:
Si, anda hermoso :+1: 
```
irb(main):002:0> Finding.includes(:finding_relations).order('finding_relations.related_finding_id DESC').order_values
=> ["finding_relations.related_finding_id DESC"]
```

Reply:
No me preocupan para nada los ```findings.algo``` yo hablo de: https://github.com/cirope/mawidabp/blob/master/app/models/concerns/findings/sort_columns.rb#L24 (o potenciales futuros de tablas "foráneas").

Reply:
Si explota hermoso, porque hace un `where().order()` sin include... ahora veo que magia tiro